iosapp/app/Views/ButtonScreenView.swift

```diff
@@ -0,0 +1,292 @@
+import SwiftUI
+
+struct ButtonScreenView: View {
+  @Environment(\.colorScheme) var cs
+
+  typealias ButtonConfig = (text: String, type: BigButton.ButtonType, true: Bool)
```

Comment:
i wonder how much it would bloat the code to convert this to a struct with named fields. i'm still not 100% clear what the Bool is here (or why it's labeled `true:`)